### PR TITLE
add css linting

### LINKED ISSUE #530: Add CSS linter
The current css code is not really styled well. We should have a linter to format it properly.

### PR DESCRIPTION
**Pull request by mike-robertson**
fixes #530

### CODE REVIEW

**Comment by mike-robertson** [comment]
@howardchung 
@blukai

File: .stylelintrc.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "extends": [
+    "stylelint-config-standard",
+    "stylelint-config-css-modules"
+  ],
+  "rules": {
+    "property-case": ["lower", { "severity": "warning" }],
+    "property-no-unknown": [true, { "severity": "warning" }],
+    "shorthand-property-no-redundant-values": false,
+    "indentation": "tabs"
```
Comment (blukai) [inline .stylelintrc.json]: Got warning on line 9 and 10, I guess `indentation` should be 2 🍀

**Comment by mike-robertson** [comment]
btw I didn't format the palette.css because it would make some things harder to read. We can fix that file later when we are sure which rules we care about and which we don't.

File: package.json [hunk 2]
```diff
@@ -10,6 +10,7 @@
     "build": "npm run lintnofix && npm test && npm run webpack",
     "lint": "eslint . --ext .js,.jsx --fix",
     "lintnofix": "eslint . --ext .js,.jsx",
+    "lint:css": "stylelint \"src/**/*.css\"",
```
Comment (howardchung) [inline package.json]: i prefer single quotes to escape chars, if possible

**Comment by howardchung** [comment]
Are we adding this to CI?

**Comment by mike-robertson** [comment]
I think we can. I will make the change with the quotes and add it to the build process I guess.

**Pull request merged**